spec/features/schools/participants/add_participants_spec.rb

```diff
@@ -15,8 +15,11 @@
   end
 
   scenario "Induction tutor can add new ECT participant" do
-    when_i_click_on_add_a_new_ect_or_mentor_link
-    then_i_am_taken_to_add_your_ect_and_mentors_page
+    when_i_am_taken_to_roles_page
+    and_the_page_should_be_accessible
+    and_percy_should_be_sent_a_snapshot_named "ECF roles information"
+    and_select_continue
```

Comment:
Let's keep the structure here: action -> assertion -> blank line -> action...
The first action in a set should start with `when`

```suggestion

    when_i_select_continue
```